### PR TITLE
Tabs are styled based on active component; resolved misc warnings 

### PR DESCRIPTION
**Pull request by MichelleCCodes**
# Description
- The user can easily track where they are in the application based on color of the navbar
- Warnings were resolved by adding a key when using map function 
- Warnings were resolved by removing unused lines of code 
- Comments were changed to clarify purpose of code (namely, styling in RenderHomePage)

## Type of Change
- [x] New feature 
- [x] Bug Fix

## CheckList: 
- [x]  No duplicate code left within changed files
- [x]  Size of pull request kept to a minimum
- [x]  Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/Home/RenderHomePage.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ import AddFaq from '../AdminTools/AddFaq';
 import ManageFaqPage from '../AdminTools/ManageFaq';
 import EditFaqPage from '../AdminTools/EditFaq';
 
-// self-import font for MUI components
+// Global styling for MUI components
```
Comment (katieolson84) [inline src/components/pages/Home/RenderHomePage.js:38]: Yes! I love to see this here so we can reuse code.

File: src/components/pages/CaseOverview/CaseOverview.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React, { useState, useEffect } from 'react';
-import { useParams, useHistory } from 'react-router-dom';
+import { useParams } from 'react-router-dom';
```
Comment (paulstgermain) [inline src/components/pages/CaseOverview/CaseOverview.js:2]: Nice removal of unused methods - it's a small thing, but cleanup is important!

File: src/components/pages/Home/RenderHomePage.js (hunk 1 above)
Comment (paulstgermain) [reply src/components/pages/Home/RenderHomePage.js:38]: Seconding this, it's definitely a nice-to-have!

File: src/components/pages/SideDrawer/SideDrawer.js [hunk 3]
```diff
@@ -85,7 +80,7 @@ export default function SideDrawer(props) {
       'resize',
       () => {
         const mobile = window.innerWidth < 769;
-        if (mobile !== mobile) {
+        if (!mobile) {
```
Comment (katieolson84) [inline src/components/pages/SideDrawer/SideDrawer.js:83]: Love the redo on this line. DRY!

Comment (paulstgermain) [inline src/components/pages/SideDrawer/SideDrawer.js:83]: Nice reworking of the original statement, this feels way less clunky!

**Review by katieolson84 — APPROVED** [review]
Your code looks great. I do not see any changes that need to be made. I will go in and help update the icons now! Good work!

**Review by paulstgermain — APPROVED** [review]
Implemented code looks great, and I love seeing the focus on cleaning the code up as we go - great work!!

**Pull request merged**